code/modules/antagonists/changeling/powers/regenerate.dm

```diff
@@ -39,3 +39,68 @@
 		H.restore_blood()
 		H.remove_all_embedded_objects()
 	return TRUE
+
+/datum/action/changeling/limbsnake
+	name = "Chimera"
+	desc = "We turn our limbs into an autonomous snake. The poison of this creatures can paralyze attackers. Costs 10 chemicals."
+	helptext = "We reform one of our limbs as an autonomous snake-like creature. This grotesque display may ward off attackers, and the creature will inject them with incapacitating poison."
+	button_icon_state = "last_resort"
+	chemical_cost = 10
+	req_absorbs = 3
```

Comment:
requiring absorbs for abilities is a little cringe and encourages murderbone. (Although saying that I might be reading this wrong and stealth stinging probably counts towards this)